packages/homepage/src/screens/home/hero/Content.js

```diff
@@ -10,6 +10,28 @@ import angular from '../../../assets/icons/home-angular.svg';
 import html from '../../../assets/icons/home-html.svg';
 import more from '../../../assets/icons/home-more.svg';
 
+import { ExperimentValues, useExperimentResult } from '@codesandbox/ab';
+
+const experimentPromise = useExperimentResult('hp-cta-iteration-1');
```

Comment:
@tromika all these hooks (useExperimentResult, useState and useEffect) should be inside of the react component to work correctly.